### PR TITLE
Make the time between cycling armors configurable with cycle_armors_time

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

**Comment by Sarvatt** [comment]
I would like to increase the default time here, but kept it functionally the same as it is now for this pull request and will send another to paint that bikeshed another color.

File: combat-trainer.lic [hunk 1]
```diff
@@ -19,7 +19,9 @@ class SetupProcess
     echo("  @priority_defense: #{@priority_defense}") if $debug_mode_ct
     @cycle_armors = settings.cycle_armors
     echo("  @cycle_armors: #{@cycle_armors}") if $debug_mode_ct
-    @last_cycle_time = Time.now - 125
+    @cycle_armors_time = settings.cycle_armors_time || 125
```
Comment (Sarvatt) [inline combat-trainer.lic]: the || 125 will be removed in a followup, I just wanted to be extra safe for now to not break people.

Comment (Sarvatt) [reply combat-trainer.lic]: actually it should probably go now? shouldn't be needed

**Comment by rpherbig** [comment]
Do you also need to change `combat-trainer`, line 106?

**Comment by Sarvatt** [comment]
What a doofus, thanks for catching that!

**Pull request merged**